src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let styles = sourceString.replace(/\n*/g, '').split(';');
+
+  styles.forEach((el, i) => {
+    styles[i] = el.trim();
+
+    if (styles[i].length === 0) {
+    }
+  });
+  styles = styles.filter(el => el.length > 0).map(el => el.split(':'));
+
+  const result = {};
+
+  styles.forEach(el => {
+    result[el[0].trim()] = el[1].trim();
+  });
```

Comment:
`el` could be destructured:
```suggestion
  styles.forEach(el => {
    const [key, value] = el;
    result[key.trim()] = value.trim();
  });
```

Reply:
yes, thank you. good point!